src/view/concept-card.inc

```diff
@@ -183,7 +183,7 @@
 
   {% else %}
   <div class="alert alert-danger">
-    <p>Error: Term "{{ term }}" not found in vocabulary!</p>
+    <p>{% trans %}Error: Term "{{ term }}" not found in vocabulary!{% endtrans %}</p>
```

Comment:
I think you should use `"%term%"` here instead of the brace syntax (looking at other similar strings that use variable interpolation)

Reply:
Ok. I also fixed this in "Description of the changes in this PR":
trans filters added to Twig templates and inc files. Translation files containing variables in their identifiers are translated with notation {% trans %} ... %variable% ... {% endtrans %}